userspace/libsinsp/examples/util.cpp

```diff
@@ -20,9 +20,13 @@ limitations under the License.
 //
 // Get the string representation of a ppm_event_category
 //
-std::string get_event_category(ppm_event_category category)
+std::string get_event_category_name(ppm_event_category category)
 {
-    switch(category)
+    /* Should be enforced by the caller but just to be sure */
+    const int bitmask = EC_SYSCALL - 1;
```

Comment:
We don't need this since the category is coming from `ev->get_category()` :)

Reply:
yep that's exactly what the comment says, this work because the caller performs the right action, i added the check also here just to be future proof

Reply:
i've removed it since we use it only in tests :)